src/main/java/org/spongepowered/api/event/command/MessageSinkEvent.java

```diff
@@ -40,21 +44,21 @@
      *
      * @return The original message sink to send to
      */
-    MessageSink getOriginalSink();
+    Optional<MessageSink> getOriginalSink();
```

Comment:
Why do you also make the message sink optional? Unlike for Text there is `MessageSinks.toNone()` which should represent the same.